cdci_data_analysis/analysis/instrument.py

```diff
@@ -295,6 +304,7 @@ def run_query(self, product_type,
                     query_out.set_failed(product_type, logger=logger, sentry_client=sentry_client, excep=e)
 
         # adding query parameters to final products
+        # TODO: this can be misleading since it's the parameters actually used
         query_out.set_analysis_parameters(par_dic)
```

Comment:
Yes, true. An alternative approach could be to call this before the rest of the default values is set. But, I would not change it now.